Gemfile

```diff
@@ -4,7 +4,34 @@ source 'https://rubygems.org'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.0'
```

Comment:
This reminds me! In the new project use the latest stable version of Rails. At time of writing this is 4.2.1. There's a few places to figure the latest stable version out from, including:
- http://weblog.rubyonrails.org/releases/
- https://rubygems.org/gems/rails/versions